### ISSUE TITLE
Avoid CORS problems when loading POI data

### ISSUE THREAD

**Issue by eggyknap** [EndPointCorp/com.endpoint.lg.webctl#3]
https://github.com/EndPointCorp/com.endpoint.lg.panoviewer/commit/789d325fc3f7632eced725114c8695e2eb3c4145 adds a sort of "proxy" in the pano viewer's java code, so all new pano resources are loaded through it, and avoid cross-origin complaints in the javascript code. We might want the same thing here, for loading the POI data.

**Comment by wzin** [comment]
possibly related EndPointCorp/lg_chef#200

**Comment by eggyknap** [comment]
Yeah, related -- but I've run into times when even after proving with tcpdump that lighttpd sends those headers, I still get CORS errors. This proxy stuff fixes it.
